src/main/scala/viper/gobra/reporting/VerifierError.scala

```diff
@@ -231,6 +231,11 @@ case class SynthesizedAssertionFalseError(info: Source.Verifier.Info) extends Ve
   override def message: String = info.comment.reduce[String] { case (l, r) => s"$l; $r" }
 }
 
+case class GoCallPreconditionError(node: Source.Verifier.Info) extends VerificationErrorReason {
+  override def id: String = "go_call_precondition_error"
+  override def message: String = s"${node.origin.tag.trim} might not satisfy method precondition."
```

Comment:
I would change the message to something like `... might not satisfy precondition of called function or method` since this error (in my understanding) can occur for both go method and go function calls, can't it?

Reply:
yes, you are right!